pkg/nodelabels/builder_test.go

```diff
@@ -63,7 +63,8 @@ func TestBuildNodeLabels(t *testing.T) {
 			expected: map[string]string{
 				RoleLabelMaster16:       "",
 				RoleLabelControlPlane20: "",
-				RoleLabelName15:         RoleMasterLabelValue15,
+				//RoleLabelAPIServer16:    "",
```

Comment:
Did you mean to leave this commented?

Reply:
Yes. The test runs without APIServerNodes feature flag. Will remove the comment when we remove/enable the ff by defult